dpdata/openmx/omx.py

```
@@ -82,24 +64,25 @@ def load_atom(lines):
+                raise RuntimeError("Cell_Vectors does not contain enough elements.")
+            values = list(map(float, parts[:9]))
+            cell = [values[0:3], values[3:6], values[6:9]]
             cells.append(cell)
-            cell = []
+            # Checking SCF converged or not
+            for token in line.split():
+                if token.startswith("scf_conv="):
+                    try:
+                        scf_conv = int(token.split("=")[1])
+                        if scf_conv == 0:
+                            warnings.warn(f"SCF not converged!")
```

Ruff (0.15.10)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)